src/adapters/ReactSixteenAdapter.js

```diff
@@ -0,0 +1,265 @@
+/* eslint no-use-before-define: 0 */
+import React from 'react';
+import ReactDOM from 'react-dom';
+import ReactDOMServer from 'react-dom/server';
+// import TestRenderer from 'react-test-renderer';
```

Comment:
let's make sure to do a sweep and remove all unneeded commented-out code before merging :-)